service/src/main/java/bio/terra/catalog/datarepo/DatarepoService.java

```diff
@@ -108,25 +102,12 @@ public DatasetAccessLevel getRole(AuthenticatedUserRequest user, String snapshot
     }
   }
 
-  @VisibleForTesting
-  SnapshotsApi snapshotsApi(AuthenticatedUserRequest user) {
-    return new SnapshotsApi(getApiClient(user));
+  private SnapshotsApi snapshotsApi(AuthenticatedUserRequest user) {
```

Comment:
My inclination is to one-line these functions, just given how simple they are. But I am also perfectly fine with them as they are. It looks like we also do that below in SamService, so we should at least keep the approach the same.

Reply:
I had left it this way to make it easier to merge in changes from `main` since this was a long running branch. Let me try inlining and see how it looks.